tendrl/node_agent/provisioner/ceph/plugins/ceph_installer.py

```diff
@@ -0,0 +1,81 @@
+
+import json
+import logging
+import urllib3
+
+from tendrl.node_agent.provisioner.ceph.provisioner_base import\
+    ProvisionerBasePlugin
+
+LOG = logging.getLogger(__name__)
+
+
+class CephInstallerPlugin(ProvisionerBasePlugin):
+
+    _MGET = 'GET'
+    _MPOST = 'POST'
+    _CEPH_INSTALLER_API_PORT = '8181'
+
+    def __init__(
+        self,
+        provisioner_node
+    ):
+        self.http = urllib3.PoolManager()
+        self.provisioner_node = provisioner_node
+
+    def install_mon(self, mons):
```

Comment:
In install_mon and install_osd, only method name differs I feel. Can we have a common function called from both?